apps/abort/main.c

```diff
@@ -123,7 +123,7 @@ int main(int argc, char *argv[])
     bool specific_address = false;
     int argi = 0;
     unsigned int target_args = 0;
-    char *filename = NULL;
+    const char *filename = NULL;
```

Comment:
Is there any reason for which we do not do line 126 and 128 in a single statement?

Reply:
I would also like that but it seems to me that is current convention to just do it this way so I prefer to keep it that way before we choose to other wise. In that case also I would do that in separate PR.

Reply:
As a developer, it is easier to see declarations, gap, operational code, gap, return in a function, so that is the pattern that I've been using in this library.

I have also been an advocate of initializing all locals to simply avoid those bugs from uninitialized locals. However, the latest compilers are really good at warning you when they detect an uninitialized local declaration, and we're using the latest compilers to build the test code and validate a lot of it with unit tests, so we should be okay to not have to initialize all the locals (and certain other contributing developers would be happier).